packages/frontend-v2/src/libs/interfaces/membershipPlan.interface.ts

```diff
@@ -1,9 +1,12 @@
+import { IMembership } from "./membership.interface";
+
 export interface IMembershipPlan {
   id: number;
   price: number;
   description?: string;
-  type: PlanType;
+  type: number;
```

Comment:
Why are you not using the enum anymore ?

Reply:
our backend return a number

Reply:
yep but enum are can also be numbers :)